vendor/bat-native-confirmations/src/bat/confirmations/internal/static_values.h

```diff
@@ -41,6 +43,534 @@ const uint64_t kRetryAdsRewardsAfterSeconds =
 const uint64_t kRetryFailedConfirmationsAfterSeconds =
     5 * base::Time::kSecondsPerMinute;
 
+const std::map<std::string, bool> kLargeAnonymityCountryCodes = {
```

Comment:
such global objects are banned by CC :( See the pattern with a static function and `NoDestructor`

Reply:
Raised https://github.com/brave/brave-browser/issues/9164 to fix this globally in Brave Ads

Reply:
Wondering why not just use a set and use existence in the set for the boolean value?

Reply:
No good reason other than so I could use the same list as Brave ads to make it easier to keep in sync. I will raise a ticket to change to `set` for both ads and confirmations. Thanks